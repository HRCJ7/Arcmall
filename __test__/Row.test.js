import "react-native";
import React from "react";
import Row from "../components/Row";

// Note: test renderer must be required after react-native.
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<Row />).toJSON();
  expect(tree).toMatchSnapshot();
});

// These serve as integration tests for the jest-react-native preset.
it("renders the Text component", () => {
  const Text = require("Text");
  const tree = renderer
    .create(<Text/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});