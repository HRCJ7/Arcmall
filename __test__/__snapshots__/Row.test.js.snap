// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "borderLeftColor": "#327EC2",
        "borderLeftWidth": 8,
        "flex": 1,
        "flexDirection": "row",
        "height": "15%",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "flexDirection": "column",
          "height": "100%",
          "marginTop": "2%",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#506f9b",
            "flex": 1,
            "fontSize": 18,
            "fontWeight": "bold",
            "marginLeft": "10%",
            "marginTop": "3%",
          }
        }
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#3F5F8E",
            "flex": 1,
            "fontSize": 12,
            "marginBottom": "15%",
            "marginLeft": "10%",
          }
        }
      />
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#3F5F8E",
          "flex": 1,
          "fontSize": 15,
          "marginRight": "10%",
          "marginTop": "11%",
          "textAlign": "right",
        }
      }
    >
      09 July 2018
    </Text>
  </View>
</View>
`;

exports[`renders the Text component 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
/>
`;
